### DISCUSSION TITLE
How about displaying the density of SNP or genes in jbrowse V2 when the displaying size is larger than a certain range

### DISCUSSION THREAD

**Discussion by yaanlpc** [GMOD/jbrowse-components#3473] [Ideas]
Hello, I am not 100% sure if Jbrowse v2 also has the feature to display the histogram of gene and SNP density when the range is getting larger after zoom-in, see attached snapshot from V1 on our flax genome.  Wondering if there is a hidden switch to toggle of this feature, I would like to apply this useful feature on our coding gene and SNPs tracks (both were imported via the vcf files.).  Thanks very much!
![jbrowse v1 0 SNP density](https://user-images.githubusercontent.com/13814897/214361115-eff2837b-63a5-4914-85a5-ac57f52a7803.jpg)

**Comment by cmdcolin** [comment]
Hi there, indeed we do not have a concept of feature density overviews built in yet.  We currently try to detect that there is "too much data to render" but it does not do the feature density overviews. The only alternative right now would be to manually create e.g. a BigWig file that represents this data and load it as a separate track

**Reply by yaanlpc** [reply]
Hi Colin, you mean converting a vcf file to the format of BigWig? In this scenario, maybe we only need the bigWig if it can show the density when the range is wide than a cutoff.

**Reply by cmdcolin** [reply]
@yaanlpc yes, essentially e.g. converting vcf to bedgraph via https://bedtools.readthedocs.io/en/latest/content/tools/genomecov.html and then to bedGraphToBigWig or something like that. and then I agree, it could show density at a certain cutoff optionally. if interested, can subscribe to this issue for more details https://github.com/GMOD/jbrowse-components/issues/463
